econml/dml.py

```diff
@@ -216,21 +236,26 @@ def fit(self, X, T_res, Y_res):
 
                 self._model.fit(cross_product(self._featurizer.fit_transform(X), T_res), Y_res)
 
-            def predict(self, X):
+            def predict(self, X, T_res=None):
```

Comment:
Is this extra argument necessary?  How does this differ from multiplying the result of the existing `predict` method (which returns a `d_y × d_t` matrix for each row) by the residuals `T_res`?  I think the calling code should probably just perform that multiplication instead of changing this method to take an extra argument.

Reply:
Yes you are right. I tried your suggestion and it gives the same scores.